docs/guides/deep_linking/dl_android_overview/dl_android_unified_deep_linking.md

```diff
@@ -88,6 +89,60 @@ The `DeepLink `object contains the deep linking information and helper functions
 >        }
 >```
 
+### Returned parameters in UDL
+
+Apps using Unified Deep Linking (UDL) receive different sets of parameters depending on whether the flow is **Deferred Deep Linking (DDL)** or **Direct Deep Linking (DL)**.
+
+#### Deferred deep linking (DDL)
+
+The app receives only the parameters relevant for deferred deep linking:
+
+- `deep_link_value`  
+- `deep_link_sub1`–`deep_link_sub10`  
+  Any other parameters return `null`.
+
+#### Direct deep linking (DL)
+
+**Deep linking parameters**: 
+
+- `deep_link_value`  
+- `deep_link_sub1`–`deep_link_sub10`  
+
+**And additional parameters**: 
```

Comment:
@dori-af This is just a partial list of the param we can return.
We need to word it again.